caas/kubernetes/provider/k8s.go

```diff
@@ -560,9 +565,9 @@ func (k *kubernetesClient) deleteSecrets(appName string) error {
 // application exists, and whether the operator is terminating.
 func (k *kubernetesClient) OperatorExists(appName string) (caas.OperatorState, error) {
 	var result caas.OperatorState
-
-	statefulsets := k.client().AppsV1().StatefulSets(k.namespace)
-	operator, err := statefulsets.Get(k.operatorName(appName), v1.GetOptions{IncludeUninitialized: true})
+	operatorName := k.operatorName(appName)
```

Comment:
This isn't the right place for this code.
We have a method Operator() which returns info about an operator.
This Exists() method is light weight and only used to see if an operator is dying.